concepticondata/references/references.bib

```diff
@@ -3390,3 +3390,28 @@ @Phdthesis{Pepper2020
   Year                                = {2020},
   Address                             = {Oslo}
 }
+
+@Article{Loukotka1944,
```

Comment:
Did we SEE this article? I haven't, and if not, this needs to be mentioned in the NOTE. That we did not see the original.

Reply:
I found the actual article. I'll send you the pdf.